distribution/solr-distro/src/main/resources/bin/solr.cmd

```diff
@@ -300,7 +300,7 @@ goto done
 :script_usage
 @echo.
 @echo Usage: solr COMMAND OPTIONS
-@echo        where COMMAND is one of: start, stop, restart, healthcheck, create, create_core, create_collection, delete, version, zk, auth, assert, config
+@echo        where COMMAND is one of: start, stop, restart, status, healthcheck, create, create_core, create_collection, delete, version, zk, auth, assert, config
```

Comment:
The  `status` command is also not on the Solr 7.4 version of the `solr.cmd` file. Was this an oversight of the Solr project team?

Reply:
I'm not sure. The command works, though!

Reply:
The `solr.cmd` file from the official Solr 7.4 release has some changes that are not on our version of the file because Solr 7.4 was released after we added the security manager stuff. Maybe we should grab the 7.4 `solr.cmd` file and re-apply our security manager changes to it? What do you think @mcalcote @garrettfreibott?

Reply:
@ahoffer Did you think there should be more changes here or can I try to get this one through?